src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -331,6 +343,16 @@ private void HideSection()
 			}
 		}
 
+		private async void ReorderItems()
+		{
+			var dialog = new ReorderSidebarItemsDialogViewModel();
+			var dialogService = Ioc.Default.GetRequiredService<IDialogService>();
+			App.Logger.Warn("Reorder items dialog popup");
```

Comment:
Why is it logged as a warning? I think it is supposed to be an `Info` log

Reply:
It was just there for testing, i'll remove it altogether